app/assets/v2/js/pages/onboard.js

```diff
@@ -229,14 +249,35 @@ $('.search-area input[type=text]').keypress(function(e) {
   }
 });
 
+const save_job_status = function() {
+  if (!$('.navbar #navbarDropdown').html()) {
+    _alert('No profile', 'error');
+  }
+  const job_search_status = $('#jobStatus').find(':selected').val();
+  const show_job_status = $('#showJobStatus').prop('checked');
```

Comment:
A minor thing but I notice if I come back to the onboarding is not showing previous election